scripts/eslint-plugin-fig/index.js

```diff
@@ -0,0 +1,60 @@
+const rmRfRegexp = /rm \-(?:f(?:(?: \-)?r)?|r(?: \-f|f)?)/;
+
+module.exports = {
+    rules: {
+        'no-malicious-script': {
+            meta: {
+                type: 'problem',
+            },
+            create: function (context) {
+                return {
+                    Property(node) {
+                        if (node.key.name === 'script') {
+                            const scriptValue = node.value.value;
+                            if (rmRfRegexp.test(scriptValue)) {
+                                context.report({
+                                    node,
+                                    message: 'No Malicious Script allowed',
+                                });
+                            }
+                        }
+                    },
+                };
+            },
+        },
+        'no-name-equals': {
+            meta: {
+                type: 'problem',
+                fixable: 'code',
+            },
+            create: function (context) {
+                return {
+                    Property(node) {
+                        if (node.key.name === 'name') {
+                            const value = node.value;
+                            if (value.type === 'Literal' && value.value.endsWith('=')) {
+                                context.report({
+                                    node,
+                                    message: 'The name property must not include `=`',
+                                    fix: function (fixer) {
+                                        const [, end] = value.range;
+                                        return fixer.replaceTextRange([end - 2, end - 1], '');
+                                    },
+                                });
+                            }
```

Comment:
Hard to read with all theses `value` variables, maybe rename the first one to add clarity:
```suggestion
                            const currentNode = node.value;
                            if (currentNode.type === 'Literal' && currentNode.value.endsWith('=')) {
                                context.report({
                                    node,
                                    message: 'The name property must not include `=`',
                                    fix: function (fixer) {
                                        const [, end] = currentNode.range;
                                        return fixer.replaceTextRange([end - 2, end - 1], '');
                                    },
                                });
                            }
```